.readthedocs.yaml

```diff
@@ -0,0 +1,40 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
```

Comment:
Can we have Fedora here as well?

Reply:
Only Ubuntu is supported, https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os